dev/Prototypes/ItemsViewPrototype/Control/ViewDefinitions/Table/TableCell.cs

```diff
@@ -71,7 +71,6 @@ protected override void OnPointerCanceled(PointerRoutedEventArgs e)
         protected override void OnPointerCaptureLost(PointerRoutedEventArgs e)
         {
             m_isPressed = false;
-            m_isPointerOver = false;
```

Comment:
>m_isPointerOver = false; [](start = 12, length = 24)

This seems suspect to me

Reply:
why? As soon as you clicked into the filter TextBox, within the table cell, OnPointerCaptureLost is called, which then causes the m_isPointerOver to go to false, and the visual state makes the TextBox that you just clicked in go to Collapsed.

Reply:
Because PointerCaptureLost can occur for a number of reasons and you won't necessarily receive a different pointer event to inform you that the pointer is not longer over your UI. It sounds like the Visual state transitions needs a more robust trigger to account for your scenario, rather than changing the m_isPointerOver boolean to reflect something other than if the pointer is over.